x-pack/plugins/monitoring/public/components/logstash/pipeline_viewer/views/detail_drawer.js

```diff
@@ -20,7 +20,8 @@ import {
   EuiFlexGroup,
   EuiFlexItem,
   EuiButtonIcon,
-  EuiSpacer
+  EuiSpacer,
```

Comment:
I'm fine with merging these changes to `DetailDrawer` to my PR, and they look fine to me, but I'd like @ycombinator to review this code as well before we're ready to merge to Kibana `master`.